odl/tomo/backends/astra_cuda.py

```diff
@@ -233,8 +236,14 @@ def _call_forward_real(self, vol_data, out=None, **kwargs):
             if self.geometry.ndim == 2:
                 out[:] = self.proj_array
             elif self.geometry.ndim == 3:
-                out[:] = np.swapaxes(self.proj_array, 0, 1).reshape(
-                    self.proj_space.shape)
+                # TODO: Find a way not to have to do rollaxis(0, 3) for
```

Comment:
Should we use moveaxis instead? It is recommended in the docs:
https://numpy.org/doc/stable/reference/generated/numpy.rollaxis.html